src/QMCWaveFunctions/Fermion/DeterminantAllocators.hpp

```diff
@@ -0,0 +1,41 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2021 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Lab
+//
+// Refactored from: OMPallocator.hpp
+//////////////////////////////////////////////////////////////////////////////////////
+// -*- C++ -*-
+/** @file
+ *  These allocators are problematic to maintain consistency of
```

Comment:
I agree it belongs in Platforms. I will rewrite the docs and move it up where it belongs. Since only the DiracBatchedDeterminant uses this so far and I made it only general enough to support my work.

This is only part of the solution to a working ENABLE_OFFLOAD=0 ENABLE_CUDA=1.

Reply:
The documentation of this is hopefully coherent now.